### DISCUSSION TITLE
Combine contextualized embeddings with static vectors for NER

### DISCUSSION THREAD

**Discussion by evabuehlmann** [explosion/spaCy#8658] [Help: Coding & Implementations] [answered]
Hi, 
we are currently training a NER model with the spacy [transformer, ner] pipeline, using pretrained BERT embeddings. 
Is it possible (and reasonable) to combine the BERT embeddings with static vectors (e.g. tok2vec) for the NER model?

Thanks! 
Eva

**Comment by Pandalei97** [comment, marked as answer]
The transformer layer replaces the tok2vec layer for the moment, you can't combine the two layers for the same NER model.
The spacy's tok2vec don't take your raw static vectors as output but will generate a contextualized embedding like your transformer model. In the most of the cases (in my experience), transformers works better than tok2vec.
That's to say, if you need to do similarity comparisons, you can still include your static vectors. Otherwise it's not really useful.
